analytic_secondaxis/__openerp__.py

```diff
@@ -32,46 +34,47 @@
 {
     "name": "Account Analytic Second Axis",
     "version": "1.1",
-    "author": "Camptocamp,Odoo Community Association (OCA)",
+    "author": "Camptocamp,Taktik,Odoo Community Association (OCA)",
     "category": "Generic Modules/Accounting",
     "description":
-    """
-    Add a second analytical axis on analytic lines allowing you to make
-    reporting on.
+        """
```

Comment:
Extract this into README.rst file and use OCA's template (https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst)